src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/OverseasEntitiesService.java

```diff
@@ -87,30 +87,11 @@ SubmissionType getSubmissionType(String requestId, String overseasEntityId) thro
         }
     }
 
-    public ResponseEntity<Object> createOverseasEntityWithResumeLink(Transaction transaction,
-                                                                     OverseasEntitySubmissionDto overseasEntitySubmissionDto,
-                                                                     String requestId,
-                                                                     String userId) throws ServiceException {
-        ApiLogger.debugContext(requestId, "Called createOverseasEntityWithResumeLink(...)");
-
-        return createOverseasEntity(transaction, overseasEntitySubmissionDto, requestId, userId, true);
-    }
-
     public ResponseEntity<Object> createOverseasEntity(Transaction transaction,
-                                                       OverseasEntitySubmissionDto overseasEntitySubmissionDto,
-                                                       String requestId,
-                                                       String userId) throws ServiceException {
-        ApiLogger.debugContext(requestId, "Called createOverseasEntity(...)");
-
-        return createOverseasEntity(transaction, overseasEntitySubmissionDto, requestId, userId, false);
-    }
-
-    private ResponseEntity<Object> createOverseasEntity(Transaction transaction,
                                                         OverseasEntitySubmissionDto overseasEntitySubmissionDto,
                                                         String requestId,
-                                                        String userId,
-                                                        boolean addResumeLinkToTransaction) throws ServiceException {
-        ApiLogger.debugContext(requestId, "Called createOverseasEntity(...), with 'add resume link' = " + addResumeLinkToTransaction);
+                                                        String userId) throws ServiceException {
+        ApiLogger.debugContext(requestId, "Called createOverseasEntity(...), with 'add resume link'");
```

Comment:
I think the _, with 'add resume link'_ part can be removed from the message.

Reply:
Removed